src/OrchardCore.Modules/OrchardCore.Contents/Recipes/ContentStep.cs

```diff
@@ -43,8 +43,9 @@ public async Task ExecuteAsync(RecipeExecutionContext context)
                     await _contentManager.CreateAsync(contentItem);
                     
                     // Overwrite ModifiedUtc & PublishedUtc values that handlers have changes
-                    context.ContentItem.ModifiedUtc = contentItem.ModifiedUtc;
-                    context.ContentItem.PublishedUtc = contentItem.PublishedUtc;
+                    // Should not be necessary if IContentManager had an Import method
```

Comment:
I just fix this by adding `Import` to `IContentManager`, please hold on, I need to write a unit test to ensure that everything work as expected

Reply:
No please. This is a much more complex task, that needs some serious design. It was removed from O1 for specific reasons.

Reply:
OK, it's fine

Reply:
At least the CI should pass